_harp/css/components/sections/_main-preview.scss

```diff
@@ -21,6 +22,7 @@
 #preview-player {
   margin: 0 auto 30px;
   width: 100%;
+  padding-top: $preview-video-ratio * 100%;
```

Comment:
Not sure what you mean? Like `vjs-fluid`?
The main reason why it's here is that without it, the player is sized to either height `150px` or `0` which causes a terrible flicker. So, if we make the padding-top it'll get sized correctly. I guess we only need to apply it up until it videojs has loaded.